cms-plugin/src/main/resources/spring/plugins/jacms/apsadmin/content/bulkCommandConfig.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:context="http://www.springframework.org/schema/context"
+        xmlns:jee="http://www.springframework.org/schema/jee"
+        xmlns:util="http://www.springframework.org/schema/util"
+        xmlns:aop="http://www.springframework.org/schema/aop"
+        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+			http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
+			http://www.springframework.org/schema/jee http://www.springframework.org/schema/jee/spring-jee.xsd
+			http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
+			http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop.xsd" >
+
+    <bean id="jacmsBaseContentBulkCommand" abstract="true"
+            class="org.entando.entando.plugins.jacms.apsadmin.content.bulk.commands.BaseContentPropertyBulkCommand" >
+        <property name="applier" ref="jacmsContentManager" />
+        <property name="contentAuthHelper" ref="jacmsContentAuthorizationHelper" />
+    </bean>
+
+    <bean id="jacmsJoinCategoryBulkCommand" parent="jacmsBaseContentBulkCommand" scope="prototype"
+            class="org.entando.entando.plugins.jacms.apsadmin.content.bulk.commands.JoinCategoryBulkCommand" />
+
+    <bean id="jacmsRemoveCategoryBulkCommand" parent="jacmsBaseContentBulkCommand" scope="prototype"
+            class="org.entando.entando.plugins.jacms.apsadmin.content.bulk.commands.RemoveCategoryBulkCommand" />
+    <!--
+    <bean id="jacmsJoinGroupBulkCommand" parent="jacmsBaseContentBulkCommand" scope="prototype"
+              class="org.entando.entando.plugins.jacms.apsadmin.content.bulk.commands.JoinGroupBulkCommand" >
```

Comment:
Can we remove these commented lines?

Reply:
there was a comment in the original code in entando-ps, i will remove it

Reply:
done